package.json

```diff
@@ -4,8 +4,8 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "npm run pregenerate && next dev",
-    "start:prod": "npm run pregenerate && node --max-old-space-size=3072 node_modules/.bin/next -p 8002",
+    "start": "npm run pregenerate && next dev -p 8000",
```

Comment:
The CORS config on the Delta-DAO servers only have a rule for localhost:8000

Reply:
No they don't they also include our portals, so do not remove please